services/web/server/src/simcore_service_webserver/projects/_nodes_handlers.py

```diff
@@ -469,23 +470,55 @@ async def get_project_services_access_for_gid(request: web.Request) -> web.Respo
         ]
     )
 
+    # Initialize groups to compare with everyone group ID
+    groups_to_compare = {EVERYONE_GROUP_ID}
+
+    # Get the group from the provided group ID
+    _sharing_with_group: Group | None = await get_group_from_gid(
+        app=request.app, gid=query_params.for_gid
+    )
+
+    # Check if the group exists
+    if _sharing_with_group is None:
+        msg = f"Group ID does not exist: {query_params.for_gid}"
+        raise ValueError(msg)
```

Comment:
where is this handled? Otherwise all unhandled errors will result in a 500! The logic here suggests that the gid provided by the client does not qualify, therefore it should be a client error (4XX). 

Perhaps a https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

Reply:
Sorry I forgot about this one, its fixed now :+1: